language/de.json

```diff
@@ -88,7 +88,7 @@
       ]
     },
     {
-      "label": "Bezeichnungen und Beschriftungen",
+      "label": "Bedienoberfläche",
```

Comment:
Just a note: The label "Bezeichnungen und Beschriftungen" felt clumsy, I agree. However, it described what was in the group. "Bedienoberfläche" is way more general and will conflict with content types that for instance have a dedicated group for visuals. "Bedienoberfläche" covers way more than just the common fields with phrases that can be translated.